pom.xml

```diff
@@ -214,9 +214,6 @@
             <resource>
                 <directory>src/main/resources</directory>
             </resource>
-            <resource>
-                <directory>target/metamodel</directory>
```

Comment:
Oh thats a good one. #TODO. Can you make an issue for it so I can investigate later? Thanks!